### PR TITLE
[pt]: Updated PT-PT translations

### PR DESCRIPTION
**Pull request by helderneves91**
Hello, updated PT-PT translations.
(Accidentally deleted old fork lol)

### CODE REVIEW

**Comment by andrey-helldar** [comment]
@jorgercosta, can you check this PR?

File: locales/pt/json.json [hunk 1]
```diff
@@ -307,16 +306,14 @@
...
+    "Having second thoughts about cancelling your subscription? You can instantly reactive your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Está com dúvidas sobre o cancelamento da sua subscrição? Pode reativar instantaneamente a sua subscrição a qualquer momento até ao final do seu ciclo de faturação atual. Após o término do seu ciclo de faturação atual, poderá escolher um plano de subscrição totalmente novo.",
     "Heard Island & Mcdonald Islands": "Ilhas Heard e McDonald",
     "Heard Island and McDonald Islands": "Ilha Heard e Ilhas McDonald",
     "Hello!": "Olá!",
     "Hide Content": "Esconder O Conteúdo",
     "Hold Up!": "Espera!",
     "Holy See (Vatican City State)": "Vatican City",
-    "Honduras": "Honduras",
```
Comment (andrey-helldar) [inline locales/pt/json.json:318]: Keys do not need to be deleted. When translations match, they are simply specified in the "excludes" file.

File: locales/pt/json.json [hunk 2]
```diff
@@ -307,16 +306,14 @@
...
-    "Hong Kong": "Hong Kong",
```
Comment (andrey-helldar) [inline locales/pt/json.json:319]: Keys do not need to be deleted. When translations match, they are simply specified in the "excludes" file.

File: locales/pt/json.json [hunk 3]
```diff
@@ -757,9 +754,8 @@
     "Turks and Caicos Islands": "Ilhas Turcas e Caicos",
     "Tuvalu": "Tuvalu",
     "Two Factor Authentication": "Autenticação de dois fatores",
-    "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application or enter the setup key.": "A autenticação de dois factores está agora ativa. Digitalize o seguinte código QR utilizando a aplicação autenticadora do seu telemóvel ou introduza a chave de configuração.",
+    "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application or enter the setup key.": "A autenticação de dois fatores está agora ativa. Digitalize o seguinte código QR utilizando a aplicação autenticadora do seu telemóvel ou introduza a chave de configuração.",
     "Type": "Tipo",
-    "Uganda": "Uganda",
```
Comment (andrey-helldar) [inline locales/pt/json.json:762]: Keys do not need to be deleted. When translations match, they are simply specified in the "excludes" file.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by andrey-helldar** [comment]
@helderneves91, can you fix that?

**Comment by helderneves91** [comment]
> @helderneves91, can you fix that?

Hi, sure, I'll handle this tonight

**Comment by andrey-helldar** [comment]
@jorgercosta is probably pretty busy, so I'll be able to accept this PR without checking it as soon as the test issues are fixed.

**Comment by helderneves91** [comment]
Hi @andrey-helldar , please validate the changes.

File: locales/pt/json.json [hunk 4]
```diff
@@ -177,15 +177,14 @@
     "Continue To Payment": "Continuar para pagamento",
     "Cook Islands": "Ilhas Cook",
     "Copy to clipboard": "Copiar para área de transferência",
-    "Costa Rica": "Costa Rica",
```
Comment (andrey-helldar) [inline locales/pt/json.json:180]: We need to get that key back

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Review by andrey-helldar — APPROVED** [review]
Obrigado!

**Comment by helderneves91** [comment]
I'm sorry but the mess, I didn't realized that we don't need to remove the excluded keys from the files. Won't happen again in the next PR's.

**Comment by andrey-helldar** [comment]
It's ok 👍

**Pull request merged**